src/utils/itemUtils.ts

```diff
@@ -0,0 +1,35 @@
+import type {
+  ItemLocationKey,
+  ItemLocation,
+  ItemLocationEndpoint,
+} from "../types/itemTypes"
+
+export const itemAvailabilityKeys = ["available", "useinlibrary"]
```

Comment:
this constant name could be a little clearer - even just by replacing availability with available. It's not immediately clear that these are the options are available.